translations/binaries/slps_strings_food2.json

```
@@ -75,49 +75,49 @@
             "string": "まんぼうの串焼き[END]",
             "original_bytes_length": 17,
             "available_bytes_size": 23,
-            "translation": "망둥어 꼬치구이[END]"
+            "translation": "개복치 꼬치구이[END]"
         },
         {
             "offset": 1816304,
             "string": "魚鍋[END]",
             "original_bytes_length": 5,
             "available_bytes_size": 7,
-            "translation": "생선전골[END]"
+            "translation": "생선 전골[END]"
         },
         {
             "offset": 1816312,
             "string": "肉鍋[END]",
             "original_bytes_length": 5,
             "available_bytes_size": 7,
-            "translation": "육전골[END]"
+            "translation": "고기 전골[END]"
         },
         {
             "offset": 1816320,
             "string": "フルーツとうふ[END]",
             "original_bytes_length": 15,
             "available_bytes_size": 15,
-            "translation": "과일 두부[END]"
+            "translation": "프루츠 두부[END]"
         },
         {
             "offset": 1816336,
             "string": "フルーツチーズ[END]",
             "original_bytes_length": 15,
             "available_bytes_size": 15,
-            "translation": "과일 치즈[END]"
+            "translation": "프루츠 치즈[END]"
         },
         {
             "offset": 1816352,
             "string": "フルーツパフェ[END]",
             "original_bytes_length": 15,
             "available_bytes_size": 15,
-            "translation": "과일 파르페[END]"
+            "translation": "프루츠 파르페[END]"
         },
         {
             "offset": 1816368,
             "string": "舟盛り[END]",
             "original_bytes_length": 7,
             "available_bytes_size": 7,
-            "translation": "배접시[END]"
+            "translation": "후나모리[END]"
```

오야코돈과 규동을 번역해서 적은 것처럼 후나모리도 '배에 담은 요리' 정도로 맞추는 게 일관성 있는 것 같아요

```suggestion
            "translation": "배에 담은 요리[END]"
```